python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4369,6 +4230,10 @@ def test_df_astype_to_categorical_ordered(ordered):
         ("int16", {}),
```

Comment:
May want to programmatically define by looping through `ALL_TYPES` and then special casing `category`.